converters/toZigbee.js

```diff
@@ -349,6 +349,25 @@ const converters = {
             );
         },
     },
+    squawk: {
+        key: ['squawk'],
+        convertSet: async (entity, key, value, meta) => {
+            const state = {'system_is_armed': 0, 'system_is_disarmed': 1};
+            const level = {'low': 0, 'medium': 1, 'high': 2, 'very_high': 3};
+            const values = {
+                state: value.state,
```

Comment:
I think changing this to `armed` and passing a boolean here instead of a string (`system_is_armed`/`system_is_disarmed`) makes things easier and less error prone.

Reply:
Ok? Just followed ZCL and as the structure are identical as `warning:` I did choose this path. Do you still want me to change this?

Reply:
Its fine like this.